pkgs/tools/filesystems/ntfs-3g/default.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://tuxera.com/opensource/ntfs-3g_ntfsprogs-${version}.tgz";
```

Comment:
maybe change the source url also to github

Reply:
After further investigation, it would make more sense to continue using the source package from tuxera.com.
The package comes with pregenerated autogen files which require additional packages such as libgcrypt unconditionally.
The compilation also depends on other autotools, whose handling implies big changes to this file.

Right now it works out of the box without any additional changes, so I would actually avoid the change of the source url.

Reply:
You can give [`autoreconfHook`](https://nixos.wiki/wiki/Packaging/Quirks_and_Caveats) a try. Otherwise, let's postpone this change.

Reply:
Let's merge the security fixes first. Switching to github can be addressed in followup PRs/issues